pom.xml

```diff
@@ -66,6 +66,7 @@
 
     <modules>
         <module>dev/kyuubi-codecov</module>
+        <module>extensions/kyuubi-server/kyuubi-server-plugin</module>
```

Comment:
extensions/server/kyuubi-server-plugin is better

Reply:
let me raise a followup pr